src/py42/clients/devices.py

```diff
@@ -238,3 +238,31 @@ def get_settings(self, guid, keys=None):
         uri = u"/api/v4/device-setting/view"
         params = {u"guid": guid, u"keys": keys}
         return self._session.get(uri, params=params)
+
+    def get_agent_state(self, guid, propertyName):
+        """Gets the agent state of the device.
+            `REST Documentation <https://console.us.code42.com/swagger/index.html?urls.primaryName=v14#/agent-state/AgentState_ViewByDeviceGuid>`__
+
+            Args:
+                guid (str): The globally unique identifier of the device.
+                propertyName (str): The name of the property to retrieve.
+
+            Returns:
+                :class:`py42.response.Py42Response`: A response containing settings information.
+            """
+        uri = u"/api/v14/agent-state/view-by-device-guid"
+        params = {u"deviceGuid": guid, u"propertyName": propertyName}
+        return self._session.get(uri, params=params)
+
+    def get_agent_full_disk_access_state(self, guid):
```

Comment:
Instead of this being a separate method, should we just make the constants available at the modular level and then have the users use which constant they want plus the `get_agent_state()` method?

Reply:
I guess I'd like to know the other types of agent state to request... Is this an ever-growing list? Is there going to be a new method for each one of them? That is why I thought the importable constants might be better. I don't have good domain knowledge on these APIs though.

Reply:
This was discussed in https://github.com/code42/py42/issues/161, I'm following @alanag13's recommendation from there. 

At this time the only property name that seems to be implemented is the fullDiskAccess, although there may be more in the future.

Reply:
Oh okay, I apologize, I didn't see that.

After seeing that, perhaps `FULL_DISK_ACCESS` should be a constant at the modular _and_ get used in this function like @alanag13 suggested.

Not a biggie right now though since there is only one.